docs/feature_dynamic_macros.md

```diff
@@ -58,6 +51,15 @@ For users of the earlier versions of dynamic macros: It is still possible to fin
 	}
 ```
 
-If the LEDs start blinking during the recording with each keypress, it means there is no more space for the macro in the macro buffer. To fit the macro in, either make the other macro shorter (they share the same buffer) or increase the buffer size by setting the `DYNAMIC_MACRO_SIZE` preprocessor macro (default value: 128; please read the comments for it in the header).
+### User Hooks
+
+There are a number of hooks that you can use to add custom functionality and feedback options to Dynamic Macro feature.  This allows for some additional degree of customization. 
+
+Note, that direction indicates which macro it is, with `1` being Macro 1, `-1` being Macro 2, and 0 being no macro. 
```

Comment:
Surround `0` in backticks to match `1` and `-1`.

Reply:
Thanks!  

Just a heads up, if you hit control+G, it lets you make a suggestion that can be committed by myself or others directly on github. 

Eg:
```suggestion
Note, that direction indicates which macro it is, with `1` being Macro 1, `-1` being Macro 2, and `0` being no macro. 
```

This isn't necessary to do (as I've already addressed the issues), but may be nice to know, in the future.